pwiz_tools/Skyline/Model/Lib/NistLibSpec.cs

```diff
@@ -989,6 +1004,18 @@ private bool CreateCache(ILoadMonitor loader, IProgressStatus status, int percen
                             ThrowIOException(lineCount, Resources.NistLibraryBase_CreateCache_Unexpected_end_of_file);
                         else if (line.StartsWith(NAME, StringComparison.InvariantCultureIgnoreCase)) // Case insensitive
                             break;
+
+                        // mzVault quirks
+                        if (line.StartsWith(@"Positive scan"))
+                        {
+                            isPositive = true;
+                            mzMatchTolerance = 0.1;

```

Comment:
This could also be a named constant to make the code more self-explanatory, e.g. why 0.01 and 0.1?